woocommerce/woocommerce-android
# [Tablets Orders] Collapsing the drawer when scrolled outside

Closes: #10492


### Description

The PR adds auto collapsing of the bottom drawer when a user scrolls to the bottom of the order creation/editing screen

### Testing instructions

Try on tablets/phones
* Orders -> Order creation
* Add products or anything that makes the scroll view big
* Expand the bottom drawer
* Scroll bottom, notice that it collapses and the whole content of the view is still accessible

### Images/gif


https://github.com/woocommerce/woocommerce-android/assets/4923871/54a53ee4-b7ea-4fde-9858-38f60dcef9a7


https://github.com/woocommerce/woocommerce-android/assets/4923871/2db20a7e-cd3b-4260-9f0a-a04647ef6ca0
